src/LiveComponent/CHANGELOG.md

```diff
@@ -364,7 +370,7 @@ live_component:
 -   Unexpected Ajax errors are now displayed in a modal to ease debugging! #467.
 
 -   Fixed bug where sometimes a live component was broken after hitting "Back:
-    in your browser - #436.
+    in your browser" - #436.
```

Comment:
Would be more like this no ?

```suggestion
-   Fixed bug where sometimes a live component was broken after hitting "Back"
    in your browser - #436.
```

But i'm not 100% sure we should change items that old 😅